spec/lib/subprocessors/call_spec.rb

```diff
@@ -1,19 +1,17 @@
 require "spec_helper"
 require "ruby_parser"
+require 'helpers'
 
 module MinitestToRspec
   module Subprocessors
     RSpec.describe Call do
+      include Helpers
 
       # Returns an S-expression representing a method call.
       def exp(method_name, argument)
         s(:call, nil, method_name.to_sym, s(:str, argument.to_s))
       end
 
-      def parse(ruby)
-        RubyParser.new.parse(ruby)
-      end
```

Comment:
I think I'd rather just keep this here for now. This level of duplication does not bother me much, especially in tests, and I don't like mixins. If we find ourselves duplicating this small bit of code 20 times, then let's reconsider, but for now I prefer the duplication.